src/coreclr/ToolBox/superpmi/superpmi-shim-collector/jithost.cpp

```diff
@@ -6,6 +6,20 @@
 #include "spmiutil.h"
 #include "jithost.h"
 
+// There is a single JitHost object created during the one-time initialization of the JIT (function jitStartup),
+// and shared amongst all subsequent compilations. Any calls to the getIntConfigValue/getStringConfigValue
+// APIs get recorded in a single, global MethodContext/CompileResult, and are copied to the
+// per-compilation MethodContext in the shim implementation of compileMethod (using recGlobalContext()).
+// This works because the JIT eagerly asks for all config values once, in the one-time jitStartup
+// function. If the JIT were to ask for config values later, during the per-compilation phase,
+// they would get recorded here in the global MethodContext, and copied to all subsequent
+// compilation MethodContexts. This would be incorrect. A solution would be to use a per-compilation
+// MethodContext in addition to the global MethodContext, but we have to allow for multi-threading. That is,
+// There could be multiple JIT compilations happening concurrently, so we can't just replace the global
```

Comment:
```suggestion
// there could be multiple JIT compilations happening concurrently, so we can't just replace the global
```

Reply:
Do you have a tracking issue to add ability to have config values in per-compilation MethodContext?

Reply:
I'm not sure it's worth having a tracking issue unless we actually need that functionality, which I currently believe we don't.